HOWTO.dual.softhsm2

```diff
@@ -0,0 +1,85 @@
+############################################################################
+#
+# SoftHSMv2 dual versions
+#
+# Works with CentOS 7
+#
+# The opensc rpm needs to be installed.
```

Comment:
Here, and below I would refer more generally to the "opensc package" instead of restricting it to rpm. Nowadays, we seem to have more package managers as there are OS distributions out there. (Same holds for the softhsm2 package).

Reply:
Updated in the various places